benchmarks/driver/parsing_test.py

```diff
@@ -25,196 +25,140 @@
 
 from esrally.driver import runner
 
-@pytest.mark.benchmark(
-    group="parse",
-    warmup="on",
-    warmup_iterations=10000,
-    disable_gc=True
```

Comment:
black collapes this into a single line, but we can prevent it if needed by adding a trailing comma here

Reply:
I'd personally prefer that we keep the current formatting here.

Reply:
done